bids2nda/main.py

```diff
@@ -414,6 +405,11 @@ def error(self, message):
         "output_directory",
         help="Directory where NDA files will be stored",
         metavar="OUTPUT_DIRECTORY")
+    parser.add_argument(
+        "experiment_id",
+        help="Experiment ID assigned by NDA for collection. Requred for fMRI studies",
+        metavar='EXPERIMENT_ID')
```

Comment:
hm, I wonder how we managed to upload before without this? 

so we were taking experiment id from metadata and it worked on our few uploads I participated in.

Do you know if it may be recent or could be optional?

Reply:
nevermind -- it is in the notes of README.md , so good to have it here in the code now but we would need to remove that note from README.md here.  care to do that @ljchang ?